libpimeval/src/pimSim.h

```diff
@@ -142,7 +146,12 @@ class pimSim
   pimParamsPerf* m_paramsPerf = nullptr;
   pimStatsMgr* m_statsMgr = nullptr;
   pimUtils::threadPool* m_threadPool = nullptr;
-  unsigned m_numThreads = 1;
+  unsigned m_numThreads = 0;
+  std::string m_memConfigFileName;
+  std::string m_configFilesPath;
+  bool m_initCalled = false;
+
+  static constexpr const char* m_defaultMemConfig = "DDR4_8Gb_x16_3200.ini";
```

Comment:
Can this be null by default, so that we use the default numbers in pimParamsDram ctor?

Reply:
Yes, It is updated.